framework/src/modules/dpos/dpos_module.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { BaseModule } from '../base_module';
+
+export class DPoSModule extends BaseModule {
+	public name = 'dpos';
+	public accountSchema = {};
```

Comment:
`type` is also an abstract property of `BaseModule` but not specified here. Same in other modules also.